server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java

```diff
@@ -602,7 +602,7 @@ public void run() {
       LOG.error("Error initializing metrics, metrics will not be emitted.", e1);
     }
     pausedMetrics = new PausedCompactionMetrics();
-    MetricsUtil.initializeProducers(this, pausedMetrics);
+    initServerMetrics(pausedMetrics);
```

Comment:
I think `this` was dropped accidentally. `Compactor` implements `MetricsProducer` too, so it's `registerMetrics` method is not getting called.

Reply:
fixed in 49b93b349e